app/Http/Controllers/Frontend/FlightController.php

```diff
@@ -110,8 +113,14 @@ public function bids(Request $request)
      */
     public function search(Request $request)
     {
-        $flights = $this->flightRepo->searchCriteria($request)->paginate();
-
+		$request['airline_id'] = Auth::user()->airline_id;
```

Comment:
Leave this out from the `$request`. Instead, add as a separate $where, after checking if the setting is enabled.